src/bootstrap/src/lib.rs

```diff
@@ -1668,10 +1668,23 @@ impl Build {
     /// You can neither rely on this being a copy nor it being a link,
     /// so do not write to dst.
     pub fn copy_link(&self, src: &Path, dst: &Path) {
-        self.copy_link_internal(src, dst, false);
+        self.copy_internal(src, dst, false, true);
     }
 
-    fn copy_link_internal(&self, src: &Path, dst: &Path, dereference_symlinks: bool) {
+    /// Links a file from `src` to `dst`.
+    /// Unlike, [`Build::copy_link`], this makes an actual copy, which is usually not required,
+    /// so `copy_link` should be used instead if possible.
+    pub fn copy_slow(&self, src: &Path, dst: &Path) {
```

Comment:
It's not doing anything non-ordinary than simple copying, so I don't think `cp_slow` is a good choice for naming this. Can we rename this function to `copy` and remove the `copy` part from other function names that are linking files instead of copying? Hopefully this can stop this confusion going forward.

Reply:
My train of thought here was that everything in bootstrap used `copy_link`, so `copy_link` should probably be the default picked by people and `copy_slow` is only used in one place and shouldn't really be needed, so I'd want to discourage people from using it unless they really need to. Especially given that some people might already know that `copy` existed and reach for it, not realizing that it was changed.

Reply:
`copy_slow` really seems very confusing and not correct..

Reply:
I can rename it to `copy` if you prefer :+1: